docs/02-Deploying/02-Configuration.md

```diff
@@ -1778,6 +1778,21 @@ AWS S3 Region. Leave blank to enable region discovery.
   	region: us-east-1
   ```
 
+#### Upgrades
+
+##### exit_if_migations_missing
```

Comment:
I wonder if we should make this `allow_missing_migrations` instead, so the default behavior is fleet doesn't start without running migrations.

It might be annoying to users sometimes, but it's also what should happen technically. cc @zwass

Reply:
Sure, I'm all ears.

Reply:
I like the idea of making it fail by default.

Reply:
Sounds good then, I'll reverse the logic.